integrations/operator/Makefile

```diff
@@ -98,10 +98,8 @@ manifests: crdgen controller-gen ## Generate WebhookConfiguration, ClusterRole a
 			"$${proto}"; \
 	done
 
-# Some comments have `{{xyz}}` which triggers the golang templating engine when installing the chart.
```

Comment:
Do we no longer need to satisfy this comment?
Just of sanity, we should do it.

Reply:
Yup because CRDs are now included as files and not as templates :) It's less work to do, and more robust.